mlir/lib/Dialect/Bufferization/Transforms/OneShotModuleBufferize.cpp

```diff
@@ -247,6 +247,15 @@ static func::FuncOp getCalledFunction(func::CallOp callOp) {
       SymbolTable::lookupNearestSymbolFrom(callOp, sym));
 }
 
+static FunctionOpInterface getCalledFunction(CallOpInterface callOp) {
```

Comment:
I think the above `static func::FuncOp getCalledFunction(func::CallOp callOp)` can be deleted.

Reply:
Removed. Thank you.